### DISCUSSION TITLE
Package Repository issue

### DISCUSSION THREAD

**Discussion by tavalin** [awawa-dev/HyperHDR#504] [General]
I tried to installed the new package repo on Ubuntu 22.04 and got the following error when trying to run "sudo apt update"

```
Reading package lists...
E: The repository 'https://awawa-dev.github.io jammy Release' does not have a Release file.
```

How can I check which versions of Ubuntu are supported?

**Comment by awawa-dev** [comment]
Hi
Please read the manual next time before doing anything, it is well explained in the instruction you used to setup the repo on your OS: https://awawa-dev.github.io/ Ubuntu is not natively supported yet but 22.04 can use Bullseye repo.

**Reply by tavalin** [reply]
Hi, thanks for your reply. 

Firstly I have to say I did read the instructions and noted the bit about replacing the "lsb_release" part but this is where I have to disagree that it was well-explained;it didn't have the context as to why you would replace "lsb_release" with "bookworm" or "bullseye" (which I know are Debian versions) - it was only your comment above that gave a small hint:

> Ubuntu is not natively supported yet but 22.04 can use Bullseye repo.

Then I found this link which further explained the link between Ubuntu versions and the Debian repos.

https://askubuntu.com/questions/445487/what-debian-version-are-the-different-ubuntu-versions-based-on
